doozer/doozerlib/backend/konflux_fbc.py

```diff
@@ -1236,13 +1270,31 @@ async def _start_build(
         if metadata.runtime.assembly == "stream":
             if group_name.startswith("openshift-"):
                 product_name = "ocp"
+                version = group_name.removeprefix("openshift-")
             else:
                 # eg: oadp-1.5 / mta-1.2
-                product_name = group_name.split("-")[0]
-            version = group_name.removeprefix("openshift-")
+                parts = group_name.split("-", 1)
+                product_name = parts[0]
+                version = parts[1]
             delivery_repo_names = metadata.config.delivery.delivery_repo_names
             for delivery_repo in delivery_repo_names:
-                additional_tags.append(f"{product_name}__{version}__{delivery_repo.split('/')[-1]}")
+                if group_name.startswith("openshift-"):
+                    additional_tags.append(f"{product_name}__{version}__{delivery_repo.split('/')[-1]}")
+                else:
+                    if self.major_minor_override:
+                        tag_with_version = (
+                            f"{product_name}__{version}__v{'.'.join(map(str, self.major_minor_override))}"
```

Comment:
```suggestion
                            f"{product_name}__{version}__v{'.'.join(map(str, self.major_minor_override))}"
```
@ashwindasr , looking over this again.  Won't this produce `oadp__oadp-1.5__v4.19` and not `oadp__oadp-1.5__v4.19__oadp-rhel9-operator` as mentioned in [comment](https://github.com/openshift-eng/art-tools/pull/2117#issue-3532700151)?

I'm ok with the shorter form, but want to ensure expectations and pass along accordingly for this change.